rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -207,7 +207,7 @@ namespace vk
 	{
 		if (!g_scratch_buffer)
 		{
-			// 32M disposable scratch memory
+			// 128M disposable scratch memory
 			g_scratch_buffer = std::make_unique<vk::buffer>(*g_current_renderer, 128 * 0x100000,
```

Comment:
@kd-11 Can you confirm the value to set in the comment is 128 to match with the buffer's size (128 * 0x100000)?

Reply:
Yea, didn't change the comment when updating.